tests/Makefile.am

```diff
@@ -291,3 +296,14 @@ AM_TESTS_ENVIRONMENT =								\
 	WINEPATH=$(top_builddir)/tools						\
 	WINE=$(WINE)								\
 	EXEEXT=$(EXEEXT)
+
+if USE_FUZZER
+fuzzing_fuzz_translate_SOURCES = fuzzing/fuzz_translate.c
+fuzzing_fuzz_translate_LDADD = $(top_builddir)/liblouis/liblouis.la
+fuzzing_fuzz_translate_CFLAGS = \
+	-fsanitize=address,fuzzer,undefined \
+	-fno-omit-frame-pointer \
+	$(AM_CPPFLAGS) 
+	-g -O2
+bin_PROGRAMS = fuzzing/fuzz_translate
```

Comment:
This should rather be `noinst_PROGRAMS`, since this is not a tool useful for end users